lib/docopt.rb

```diff
@@ -1,6 +1,7 @@
 module Docopt
   VERSION = '0.6.0'
 end
+
```

Comment:
This is contributed code so I would rather leave this alone.  Maybe you could suggest changes upstream instead?

Reply:
@saltedcoffii Is there a newer version than 0.6.1? (I note you're the last contributor to docopt...)

Reply:
I'm the latest contributor because someone forgot to bump the version when releasing a new version so I just did a quickfix and PR